swagger.yaml

```diff
@@ -703,6 +703,60 @@ paths:
           description: "Unexpected Cognito response"
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /groups/{id}/members?sortBy=forename:asc:
```

Comment:
I don't see this query as another endpoint as this entry suggests (is this copied from another swagger spec?)

I see the query params acting more like `limit` [here](https://github.com/ONSdigital/dp-dataset-api/blob/develop/swagger.yaml#L238) as a parameter to an existing endpoint